frontend/src/scenes/experiments/ExperimentView/Exposures.tsx

```diff
@@ -45,11 +46,11 @@ export function Exposures(): JSX.Element {
             type: 'line',
             data: {
                 labels: exposures.timeseries[0].days,
-                datasets: exposures.timeseries.map((series: Record<string, any>) => ({
+                datasets: exposures.timeseries.map((series: Record<string, any>, index: number) => ({
                     label: series.variant,
                     data: series.exposure_counts,
-                    borderColor: 'rgb(17 17 17 / 60%)',
-                    backgroundColor: 'rgb(17 17 17 / 40%)',
+                    borderColor: getSeriesColor(index),
+                    backgroundColor: getSeriesBackgroundColor(index),
                     fill: true,
```

Comment:
I would suggest to set fill to false to get a line chart, and "unstack" it ( by setting `stacked: false` below and removing the `stack: 'stack1'` it so it looks like this. I think it draws too much attention as an filled area chart. Feel free to do that in another PR or to disagree!

<img width="1719" alt="Screenshot 2025-04-11 at 11 24 46" src="https://github.com/user-attachments/assets/9e3dd7fe-e741-4513-8f03-d104f12fad3e" />


```suggestion
                    fill: true,
```